cytotable/convert.py

```diff
@@ -310,15 +404,26 @@ def _source_pageset_to_parquet(
     )
     pathlib.Path(source_dest_path).mkdir(parents=True, exist_ok=True)
 
+    # build tablenumber segment addition (if necessary)
+    tablenumber_sql = (
+        # to become tablenumber in sql select later with bigint (8-byte integer)
+        # we cast here to bigint to avoid concat or join conflicts later due to
+        # misaligned automatic data typing.
+        f"CAST({source['tablenumber']} AS BIGINT) as TableNumber, "
+        if source["tablenumber"] is not None
+        # if we don't have a tablenumber value, don't introduce the column
```

Comment:
```suggestion
        # if we already have a tablenumber value, don't introduce the column
```

Am I reading this correctly?

Reply:
Thanks for checking on this - I should be more clear here, adding tablenumber values is optional via a parameter to CytoTable. If we aren't supposed to add tablenumbers we evaluate this to an empty string (meaning we won't add a tablenumber).